src/Common/test/Common.Hosting.Test/Steeltoe.Common.Hosting.Test.csproj

```diff
@@ -7,7 +7,6 @@
     <TargetFrameworks>netcoreapp3.0;net461;netcoreapp2.1;</TargetFrameworks>
   </PropertyGroup>
   <Import Project="..\..\..\..\sharedtest.props" />
-  <Import Project="..\..\..\..\targetframework.props" />
```

Comment:
We don't need this anymore?

Reply:
3.x isn't running tests for net461. This line was removed during the merge to master and is still present in 2.x